### ISSUE TITLE
aria-flowto is not output correctly

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/standards-support#326]
### Summary

aria-flowto is not output correctly

1. Go to https://codepen.io/jaws-test/pen/KKPOvdg
2. Navigate to the link "starting point"
3. press = to display the list of all targets
4. Go to https://codepen.io/jaws-test/pen/bGbXrEd
5. Navigate to the non-interactive "starting point" 
6. press = to display the list of all targets

Note: In both examples the first 3 targets are labeled with content and the last 3 targets with `aria-label`.

### Expected result

* Consistent output
* List with all elements in `aria-flowto` is output in correct order and with their correct labels.
* List is displayed independent of the role of the starting point

### Actual result

* Different output with different browsers
* Firefox: list is not displayed, when pressing = JAWS jumps only to the first linked element
* Chrome: list is displayed in wrong order
* IE, Chrome: Some list entries are displayed without labels. Only elements with explicit labels are displayed correctly. Elements labeled with content are never (IE 11) or sometimes (Chrome for elements without role) displayed with labels.
* Edge: `aria-flowto` doesn't work at all
* Firefox, Chrome: list is only displayed if starting point is an interactive element otherwise no targets are detected

#### JAWS version and build number

JAWS 2019.1909.28
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 76.0.3809.132
Firefox ESR 60.9.0
IE 11.1331.16299.0
Edge 16.16299

**Comment by stevefaulkner** [comment]
@JAWS-test Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).

**Comment by JAWS-test** [comment]
@stevefaulkner 

Existing problems with JAWS 2024 and current Chrome:
* in the flow list, destination 2 is displayed without a label
* No possibility to jump back from the destination to the starting point
* Output of "No further flows found" when = is pressed at a destination
